packages/kit/src/core/dev/index.js

```diff
@@ -177,7 +177,15 @@ class Watcher extends EventEmitter {
 
 		this.vite = await vite.createServer(merged_config);
 
-		handler = await create_handler(this.vite, this.config, this.dir, this.cwd, this.manifest);
+		const get_manifest = () => {
+			if (!this.manifest) {
+				throw new Error("Shouldn't clear manifest after server initialized");
```

Comment:
We use single quotes for all strings
```suggestion
				throw new Error('Manifest is not available');
```